ncm-hostsfile/src/test/resources/dual_stack_localhost.pan

```diff
@@ -0,0 +1,15 @@
+object template dual_stack_localhost;
+
+prefix '/software/components/hostsfile';
+
+'active' = true;
+'file' = '/tmp/hosts.local';
+'takeover' = true;
+
+prefix 'entries';
+# IPv4
+'localhost/ipaddr' = '127.0.0.1';
```

Comment:
@jrha it might make sense to define 2 final variables with these ipv4 and ipv6 dicts, so other can use this workaround with ease.

i'm willing to merge in any case ;)